api/src/test/java/org/pmiops/workbench/api/NotebooksControllerTest.java

```diff
@@ -145,6 +146,12 @@ public void testNotebookFileList() {
     Blob mockBlob1 = mock(Blob.class);
     Blob mockBlob2 = mock(Blob.class);
     Blob mockBlob3 = mock(Blob.class);
+    FileDetail fileDetail1 = new FileDetail();
+    FileDetail fileDetail2 = new FileDetail();
+    FileDetail fileDetail3 = new FileDetail();
+    fileDetail1.setName("mockFile.ipynb");
```

Comment:
I would like to see a more explicit connection between the Blob and FileDetail mocks.  One possible approach:
```
fileDetail1.setName(NB1_NAME);
when(mockBlob1.getName()).thenReturn(NB1_NAME);
when(mockCloudStorageClient.blobToFileDetail(mockBlob1,"bucket")).thenReturn(fileDetail1);
```

Possibly this could all be in a helper method like `mockNotebookForTest(String name)`

Reply:
Good thinking. I like the idea of consolidating this logic. I ended up doing this via an inner class. Please let me know what you think.